packages/libs/sdk/README.md

```diff
@@ -0,0 +1,154 @@
+# QuickNode SDK
+
+An SDK from [QuickNode](https://www.quicknode.com/) making it easy for developers to interact with the blockchain.
+
+Currently supports getting started with [Icy Tools GraphQL API](https://developers.icy.tools/) in a blink!
```

Comment:
We should update this readme by removing ICY

Reply:
good catch, updated!